### PR TITLE
Updated version for v.2.4.0 release of beacon-apis.

### PR DESCRIPTION
**Pull request by rolfyone**
Update version for release, and referencing 1.3.0 of consensus-spec.

### CODE REVIEW

File: beacon-node-oapi.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ info:
 
     Note that it is possible for a field to be added to an endpoint's data or metadata without an increase in the version number.
 
-  version: "Dev - Eth2Spec v1.1.0"
```
Comment (rkapka) [inline beacon-node-oapi.yaml:24]: Is the name `Eth2Spec` offical? Searching for it brought me to this outdated page: https://pypi.org/project/eth2spec. And https://github.com/ethereum/consensus-specs is named `Ethereum Proof-of-Stake Consensus Specifications`. Maybe we should rename it?

Comment (rolfyone) [reply beacon-node-oapi.yaml:24]: fair point, i can probably change the name now :).

Comment (rolfyone) [reply beacon-node-oapi.yaml:24]: The drop-down is pretty short so might still be some abbreviation

**Comment by rolfyone** [comment]
@michaelsproul if not busy...

**Review by michaelsproul — APPROVED** [review]
Go for it!

**Pull request merged**